tasks/fix-cat2.yml

```diff
@@ -98,18 +115,28 @@
       - RHEL-07-010090
 
 - name: "MEDIUM | RHEL-07-010100 | PATCH | The Red Hat Enterprise Linux operating system must initiate a session lock for the screensaver after a period of inactivity for graphical user interfaces."
-  command: "true"
-  changed_when: no
+  lineinfile:
+      create: yes
+      dest: /etc/dconf/db/local.d/00-screensaver
+      insertafter: '^#?[org/gnome/desktop/screensaver]'
+      regexp: '^#?idle-activation-enabled'
+      line: idle-activation-enabled=true
+  register: rhel_07_010100
+  notify: dconfupdate
   when:
       - rhel7stig_gui
       - rhel_07_010100
   tags:
       - RHEL-07-010100
-      - notimplemented
 
 - name: "MEDIUM | RHEL-07-010101 | PATCH | The Red Hat Enterprise Linux operating system must prevent a user from overriding the screensaver idle-activation-enabled setting for the graphical user interface."
-  command: "true"
-  changed_when: no
+  lineinfile:
+      create: yes
+      dest: /etc/dconf/db/local.d/locks/session
+      regexp: '^#?idle-activation-enabled'
```

Comment:
`regexp` probably isn't necessary unless needed for cleaning up previous names of this setting, but then a separate task for that might be better.